apps/web/app/[lang]/dashboard/training-record/page.tsx

```
@@ -86,23 +86,23 @@ export default function TrainingRecordPage() {
 				<div className="flex gap-2">
 					{filter.hasOwnProperty("weight") && (
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)